translate/storage/cpo.py

```diff
@@ -575,7 +587,6 @@ def buildfromunit(cls, unit, encoding=None):
         elif isinstance(unit, pocommon.pounit):
             newunit = cls(unit.source, encoding)
             newunit.target = unit.target
-            #context
```

Comment:
Here you are removing the comment while below it is arranged properly. I prefer it to be arranged properly since seems to be communicating something useful.